docker/install.sh

```diff
@@ -40,7 +40,7 @@ sudo chgrp carma /opt/carma # Set group to expose permissions for build
 if [[ ! -z "$ROS1_PACKAGES$ROS2_PACKAGES" ]]; then
     if [[ ! -z "$ROS1_PACKAGES" ]]; then
         echo "Incrementally building ROS1 packages: $ROS1_PACKAGES"
-        colcon build --install-base /opt/carma/install --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-above $ROS1_PACKAGES 
+        colcon build --install-base /opt/carma/install --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-above $ROS1_PACKAGES --packages-skip novatel_oem7_driver novatel_oem7_msgs
```

Comment:
It looks like we are installing the ros2 version of this package, so this package skip should be in the ros2 build

Reply:
We checkout the git repository for these novatel packages, but still have to build them, so I had to skip them in the ROS 1 build since they were causing a build failure due to being ROS 2 packages. Note that I did initially commit the `--packages-skip novatel_oem7_driver novatel_oem7_msgs` line incorrectly to the ROS 1 incremental build line, but have moved it to the ROS 1 full build line instead.